ui/appui-react/package.json

```diff
@@ -132,11 +132,13 @@
     "@itwin/itwinui-react": "^2.5.0",
     "@itwin/itwinui-variables": "^2.0.0",
     "@itwin/itwinui-icons-react": "^2.1.0",
+    "@itwin/itwinui-illustrations-react": "^2.0.1",
     "@itwin/itwinui-icons": "^1.14.0",
     "classnames": "2.3.1",
     "immer": "9.0.6",
     "lodash": "^4.17.10",
-    "rxjs": "^6.6.2"
+    "rxjs": "^6.6.2",
+    "react-error-boundary": "4.0.3"
```

Comment:
do we need to pull in a separate pkg for this? 
this is what we do in the viewer https://github.com/iTwin/viewer/blob/master/packages/modules/viewer-react/src/components/error/ErrorBoundary.tsx

Reply:
I think we can stick with this instead of defining our own ErrorBoundary, just make sure to sort the dependencies alphabetically in package.json